test/.mocharc.json

```diff
@@ -6,6 +6,6 @@
   "slow": 1500,
   "timeout": 20000,
   "recursive": true,
-  "file": ["test/util.test.ts"],
+  "file": ["test/api.test.ts"],
```

Comment:
Does this need to be included?

Reply:
Thinking about it it should be `["test/**/*.test.ts"]` but yes it needs to be included, it only ran `api.test.ts` before which only called `calculateNftId`, and that's been removed.